skysurvey/target/afterglow.py

```diff
@@ -47,16 +40,28 @@
 
 template = sncosmo.Model(sncosmo.TimeSeriesSource(phases, wave, np.array(flux)))
```

Comment:
sncosmo.TimeSeriesSource requires `phases` to be in units of days, but it is currently given in seconds.  I suggest changing `phases` in this line to `phases/3600/24` or something similar

Reply:
sure. I changed it and submitted a PR again

Reply:
@AkshayPanayada Can you please copy in the current version of the file and make your changes on that to not change all of the whitespace? Additionally, please fix the rate code to as suggested in the comment above (i.e. return 0.0157+0.118*np.array(redshift_)/(1+(np.array(redshift_)/3.23)**4.66))

Reply:
sure. Will do that